package.json

```diff
@@ -283,7 +283,7 @@
       "commandPalette": [
         {
           "command": "one.backend.infer",
-          "when": "false"
+          "when": "true"
```

Comment:
After checking some tests, this will be reverted.

Reply:
Q) Will this be going to be merged and reverted after some tests in main branch?

Reply:
No. If you agree on this design, I'll add new patch to revert this on this PR.

Reply:
Done